stubs/regex/regex/_regex.pyi

```diff
@@ -0,0 +1,185 @@
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> Match[AnyStr] | None: ...
+    def split(
+        self, string: AnyStr, maxsplit: int = ..., concurrent: bool | None = ..., timeout: int | None = ...
+    ) -> list[AnyStr | Any]: ...
+    def splititer(
+        self, string: AnyStr, maxsplit: int = ..., concurrent: bool | None = ..., timeout: int | None = ...
+    ) -> Splitter[AnyStr]: ...
+    def findall(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> list[Any]: ...
+    def finditer(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> Scanner[AnyStr]: ...
+    def sub(
+        self,
+        repl: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> AnyStr: ...
+    def subf(
+        self,
+        format: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> AnyStr: ...
+    def subn(
+        self,
+        repl: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> tuple[AnyStr, int]: ...
+    def subfn(
+        self,
+        format: AnyStr | Callable[[Match[AnyStr]], AnyStr],
+        string: AnyStr,
+        count: int = ...,
+        flags: int = ...,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> tuple[AnyStr, int]: ...
+    def scanner(
+        self,
+        string: AnyStr,
+        pos: int | None = ...,
+        endpos: int | None = ...,
+        overlapped: bool = ...,
+        concurrent: bool | None = ...,
+        timeout: int | None = ...,
+    ) -> Scanner[AnyStr]: ...
+
+class Match(Generic[AnyStr]):
+
+    re: Pattern[AnyStr]
+    string: AnyStr
+    pos: int
+    endpos: int
+    partial: bool
+    regs: tuple[tuple[int, int], ...]
+    fuzzy_counts: tuple[int, int, int]
+    fuzzy_changes: tuple[list[int], list[int], list[int]]
+    lastgroup: str | None
+    lastindex: int | None
+    @overload
+    def group(self, __group: int | str = ...) -> AnyStr | Any: ...
```

Comment:
Why is the `Literal[0]` overload missing?

Reply:
added it in now 👍